apps/native-component-list/src/screens/NotificationScreen.tsx

```diff
@@ -27,6 +29,54 @@ export default class NotificationScreen extends React.Component<
     this.state = {};
   }
 
+  private _onReceivedListener: EventSubscription | undefined;
+  private _onResponseReceivedListener: EventSubscription | undefined;
+
+  componentDidMount() {
+    if (Platform.OS !== 'ios' && Platform.OS !== 'android') {
+      return;
+    }
+    // Using the same category as in `registerForPushNotificationsAsync`
+    Notifications.setNotificationCategoryAsync('welcome', [
```

Comment:
this was previously removed by me so that it doesn't conflict with `notification-tester` app, but it renders the notification testing in expo go malfunctioning, so I'm adding it back. Nothing has changed here.